task/09-functions-n-closures-tasks.js

```diff
@@ -84,7 +97,11 @@ function getPolynom() {
  *   memoizer() => the same random number  (next run, returns the previous cached result)
  */
 function memoize(func) {
-    throw new Error('Not implemented');
+    const cashedValue = func()
+
+    return function() {
+        return cashedValue
+    }
```

Comment:
```suggestion
    return () => cashedValue
```

Just another way to return the function, looks a bit shorter, but your solution is also fine